test/integration/ContractWithOverloads.spec.ts

```diff
@@ -2,7 +2,7 @@ import { expect } from "chai";
 import { deployContract } from "./utils/web3Contracts";
 
 import { web3 } from "./web3";
-import { ContractWithOverloads } from "./abis/ContractWithOverloads";
+import { __ContractWithOverloads_sol_ContractWithOverloads as ContractWithOverloads } from "./abis/__ContractWithOverloads_sol_ContractWithOverloads";
```

Comment:
isn't there a way to make solcjs generate more readable file names?

Reply:
It pollutes a lot of code otherwise

Reply:
I didn't find one. I `cd` into that directory to stop it generating even more noise. I will dig a bit more and get back to you.